### PR TITLE
removes spaces before and after link

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2016-04-06 at 09 36 29|screen shot 2016-04-06 at 09 51 06|

### CODE REVIEW

**Comment by FinalAngel** [comment]
awesome :+1:

File: CHANGELOG.txt [hunk 1]
```diff
@@ -1,12 +1,16 @@
 CHANGELOG
 =========
 
+1.1.0 (2016-xx-xx)
```
Comment (vxsx) [inline CHANGELOG.txt]: it doesn't make it 1.1.0, probably 1.0.10 :)

**Comment by vxsx** [comment]
:+1:

**Comment by vxsx** [comment]
:+1:  :+1:

**Pull request merged**